### PR TITLE
Remove the assumption on Language being found and understood

### PR DESCRIPTION
**Pull request by Jym77**
All rules with Language as input aspect had an assumption about it being determined and understood.
This assumption is moved to the Input aspects document (https://github.com/w3c/wcag-act/pull/521 even though it seems to not be published yet 🤔 ) So we can remove it from the individual rules.

This should probably not be merged before the change in input aspects got published.

Closes issue(s):

- N/A

Need for Call for Review:
This will require a 1 week Call for Review (moving an assumption to a shared document).

---

### **When creating PR:**

- [X] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [X] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [X] Add yourself (and co-authors) as "Assignees" for PR.
- [X] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [ ] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [X] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Review by WilcoFiers — APPROVED** [review]

File: _rules/video-only-as-media-alternative-fd26cf.md
Comment (daniel-montalvo) [inline _rules/video-only-as-media-alternative-fd26cf.md:39]:
```suggestion
This rule assumes that a mechanism is available to start the video and that the video element is not simply used to display the [poster](https://www.w3.org/TR/html5/semantics-embedded-content.html#element-attrdef-video-poster).
```

File: _rules/video-only-audio-track-d7ba54.md
Comment (daniel-montalvo) [inline _rules/video-only-audio-track-d7ba54.md:36]:
```suggestion
This rule assumes that a mechanism is available to start the video and that the video element is not simply used to display the [poster](https://www.w3.org/TR/html5/semantics-embedded-content.html#element-attrdef-video-poster).
```

File: _rules/video-only-description-track-ac7dc6.md
Comment (daniel-montalvo) [inline _rules/video-only-description-track-ac7dc6.md:37]:
```suggestion
This rule assumes that a mechanism is available to start the video and that the video element is not simply used to display the [poster](https://www.w3.org/TR/html5/semantics-embedded-content.html#element-attrdef-video-poster).
```

File: _rules/video-only-transcript-ee13b5.md
Comment (daniel-montalvo) [inline _rules/video-only-transcript-ee13b5.md:41]:
```suggestion
This rule assumes that a mechanism is available to start the video and that the video element is not simply used to display the [poster](https://www.w3.org/TR/html5/semantics-embedded-content.html#element-attrdef-video-poster).
```

**Review by daniel-montalvo — COMMENTED** [review]
Just these minor grammar and consistency issues for your consideration. Will approve when addressed.

**Comment by Jym77** [comment]
@daniel-montalvo All these are about existing assumptions that are not touched by this PR. As much as I agree the changes are needed, I also think that we shouldn't do them as part of this PR in order to keep things focused (the PR is already touching a lot of rules, so I'd prefer keeping it focused on doing one thing).

**Comment by daniel-montalvo** [comment]
> 
> 
> @daniel-montalvo All these are about existing assumptions that are not touched by this PR. As much as I agree the changes are needed, I also think that we shouldn't do them as part of this PR in order to keep things focused (the PR is already touching a lot of rules, so I'd prefer keeping it focused on doing one thing).

Fair enough. Let's then go ahead with as-is and keep the others for later.

**Review by daniel-montalvo — APPROVED** [review]

**Review by tbostic32 — APPROVED** [review]

**Review by carlosapaduarte — APPROVED** [review]

**Review by kherr-a11y — APPROVED** [review]

**Review by kengdoj — APPROVED** [review]

**Comment by Jym77** [comment]
Waiting for https://github.com/w3c/wcag-act/pull/521 to be published before merging this one.
It is currently in the [Editor's draft](https://w3c.github.io/wcag-act/NOTE-act-rules-common-aspects.html#input-aspects-text) version, but not in the [published one](https://www.w3.org/TR/act-rules-aspects/#input-aspects-text).

Publishing the input aspect might be delayed until #1783 is solved.

**Comment by Jym77** [comment]
The new version of the [Input aspects](https://www.w3.org/TR/act-rules-aspects/) has been published.
Merging this.

**Pull request merged**
